build.gradle

```diff
@@ -31,3 +43,25 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+detekt {
+    debug = true
+    toolVersion = "$detekt_version"
+    input = files("app/src/main/java")
+    filters = ".*/resources/.*,.*/build/.*"
+    reports {
+        xml.enabled = true
+        html {
+            enabled = true
+            destination = file("detekt/detekt.html")
```

Comment:
It would be much better to generate the output into the build folder. For example ``build/reports/detekt``.

Reply:
Ok. I reset the patch of the report.